code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -57,6 +64,23 @@
 	if(istype(loc, /obj/effect/dummy/phased_mob/slaughter))
 		bloodspell.phased = TRUE
 
+/mob/living/simple_animal/slaughter/CtrlShiftClickOn(atom/A)
+	if(!isliving(A))
```

Comment:
er, you do realize that to control+shift+click something, you have to stop in place (since holding down control causes your character to stop (voluntarily) moving across tiles), right? so you're expecting a slaughter demon to just do a full stop in the middle of combat and manage to click on someone IN MELEE while full-stopped?

that sounds about as hard as grabbing a cuffed person (who's running around like a headless chicken to try to waste your time) as a cyborg (cyborgs have to control+click people (or use the right-click menu, I guess) in order to grab them). try that some time soon; it's NOT easy.